app/javascript/modules/listingDetails/ListingDetailsEligibility.tsx

```diff
@@ -20,6 +22,43 @@ export const ListingDetailsEligibility = ({
   listing,
   imageSrc,
 }: ListingDetailsEligibilityProps) => {
+  /* TODO: Implement updated API to get actual data */
+  const HMITableHeaders = {
+    householdSize: "t.householdSize",
+    maxIncomeMonth: "t.maximumIncomeMonth",
+    maxIncomeYear: "t.maximumIncomeYear",
+  }
+  const HMITableData = [
+    {
+      householdSize: { content: `1 ${t("listings.person")}` },
+      maxIncomeMonth: { content: `$1,111 ${t("t.perMonth")}` },
+      maxIncomeYear: { content: `$51,111 ${t("t.perYear")}` },
```

Comment:
hm, perhaps we should create new text props that pass in the numbers as a param rather than concatenation